Package.swift

```diff
@@ -3,8 +3,7 @@ import PackageDescription
 
 let package = Package(
     name: "Lottie",
-    platforms: [.iOS(.v12)],
-    // platforms: [.iOS("9.0"), .macOS("10.10"), tvOS("9.0"), .watchOS("2.0")],
+    platforms: [.iOS(.v9), /*.macOS(.v10_10), */.tvOS(.v9), .watchOS(.v2)],
```

Comment:
Why is macOS commented out?

Reply:
Getting macOS works, requires changing some code changes and move things around so for now I commented it out as it wasn't also originally supported.